actors/migration/nv3/reward.go

```diff
@@ -5,11 +5,36 @@ import (
 
 	cid "github.com/ipfs/go-cid"
 	cbor "github.com/ipfs/go-ipld-cbor"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/specs-actors/actors/builtin/reward"
 )
 
 type rewardMigrator struct {
 }
 
 func (m *rewardMigrator) MigrateState(ctx context.Context, store cbor.IpldStore, head cid.Cid) (cid.Cid, error) {
-	return head, nil
+	var st reward.State
+	if err := store.Get(ctx, head, &st); err != nil {
+		return cid.Undef, err
+	}
+
+	// The baseline function initial value and growth rate are changed.
+	// As an approximation to working out what the baseline value would be at the migration epoch
+	// had the baseline parameters been this way all along, this just sets the immediately value to the
```

Comment:
nit: "the immediately value" => "the immediately successive value" or something like that